### DISCUSSION TITLE
Deploying Milvus on GCP Costs

### DISCUSSION THREAD

**Discussion by cverbah** [milvus-io/milvus#19308] [Q&A and General discussion]
Hi everybody!, Im looking for the option of using Milvus as a vector database for my company but wondering if somebody knows the monthly costs incurred in deploying it  in GCP using the same configuration shown in the Milvus docs. 
thank you very much!
Cristian

**Comment by yhmo** [comment]
Here is a milvus-on-cloud service: https://cloud.zilliz.com/login, we provide a Managed service for Milvus for 30 days free. You can have a try.
